epictrack-web/src/components/workPlan/event/EventList.tsx

```diff
@@ -495,6 +484,17 @@ const EventList = () => {
     }
   }, [ctx.selectedWorkPhase?.work_phase.phase.id]);
 
+  const getWorkPhaseById = React.useCallback(async () => {

```

Comment:
I hope this should be invoked only if the event.event_position == END

Reply:
will do that